src/docs/sphinx/dev_guide/testing.rst

```diff
@@ -72,6 +72,22 @@ Requirements:
 
    ATS also outputs both a ``.log`` and ``.log.err`` for each test and checker that is run.
 
+#. **Rebaselining tests (as needed).**
+    If tolerances to tests need to be updated, first ensure you've generated new tolerances by running the integration
+    tests like mentioned above. Then, use the ``-b`` option.
+
+    # Single baseline
+    $ ./ats.sh -b dyn_solve_serial
+
+    # Comma-separated baselines
+    $ ./ats.sh -b dyn_solve_serial,dyn_solve_parallel
+
+    # All baselines
+    $ ./ats.sh -b all
+
+    This will update the json files located in the `serac_tests <https://github.com/LLNL/serac_tests>`_ submodule. To
+    avoid Caliper files from additionally being generated, configure with ``-DENABLE_BENCHMARKS=OFF``.
```

Comment:
The caliper files issue should be resolved when we can set the `WORKING_DIRECTORY` to test in BLT. I'll start working on that this week.

https://github.com/LLNL/serac/pull/1222#discussion_r1755468490